recipes/f90nml/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "f90nml" %}
+{% set version = "0.21" %}
+{% set sha256 = "df0538bfd977f6e72d3bbf6beb8bd7dc7c01882d5a89e99fbfdfbf1aa5b24b8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/60/9d/2b784b60519009fb2b2563190e22dc8e169c7238c63caa981cf1fed15a47/{{ name }}-{{ version }}.tar.gz
```

Comment:
One last thing, can you change the URL to a more readable

`https://pypi.io/packages/source/f/f90nml/f90nml-{{ version }}.tar.gz`

It is up to you to use `{{ name[0] }}/{{ name }}/{{ name[0] }}-` too. I like only the version as a jinjs variable to make it easier to copy-n-past the URL.